hw/bsp/sensorhub/src/hal_bsp.c

```diff
@@ -69,7 +90,7 @@ hal_bsp_flash_dev(uint8_t id)
     if (id != 0) {
         return NULL;
     }
-    return &stm32f4_flash_dev;
```

Comment:
I did not expect this to still exist, does this BSP build?

Reply:
blinky builds with this BSP and changes in this PR.
Not tested though.

Reply:
A grep returns only `extern const struct hal_flash stm32f4_flash_dev;` in `hw/mcu/stm/stm32f4xx/include/mcu/stm32f4_bsp.h`, weird!

Reply:
Btw, do you have any idea what board this is?

Reply:
I have no clue, I spent few minutes to find, and failed

Reply:
Same here! ;-) I assume it might be some custom board contributed way back that might not be available anymore...